pypesto/ensemble/ensemble.py

```diff
@@ -160,6 +171,12 @@ def compute_summary(self,
         if not self.prediction_results:
             raise ArithmeticError('Cannot compute summary statistics from '
                                   'empty prediction results.')
+        # if weightings shall be used, check whether weights are there
+        if weighting:
+            if not self.prediction_results[0].conditions[0].output_weight:
+                logger.warning('WARNING: There are no weights in the '
+                               'prediction results. Will use 1 as weight.')
```

Comment:
what do you mean by that? you mean do a try... except loop? how would that benefit? this is only triggered if you actually want to use weightings but none are existent so I would not count that as a default?

Reply:
Ah I mean, because the default here is `weighting=False`, then `weighting=True` suggests the user explicitly requested weights (or is using another method that does). If weights are explicitly requested, then this suggests that the user thinks weights are available. It would then be an error (i.e. `Exception`, not warning) if weights are not available (as the user can simply provide `weighting=False` to use 1 as weights).

Reply:
ahhh yes, now I get it, very good idea :)